packages/Sandblocks-Smalltalk/SBStMethod.class.st

```diff
@@ -98,6 +98,15 @@ SBStMethod >> initialize [
 			addMorphBack: (signature := SBStSignature new))
 ]
 
+{ #category : #'as yet unclassified' }
+SBStMethod >> isAttractedTo: aMorph [
+	
+	(aMorph class == SBStClass) ifTrue: [^ self currentClass == aMorph shownClass].
+	(aMorph class == SBStMethod) ifTrue: [^ self currentClass == aMorph currentClass].
```

Comment:
here, `^ aMorph relatedClass theNonMetaClass = self relatedClass theNonMetaClass` would be the shortest formulation to get what you want I believe